usercommandmgmt/usercommandmgmt.py

```diff
@@ -0,0 +1,226 @@
+            try:
+                await ctx.send("Admin Override.")
+                await self.commandobj.delete(ctx=ctx, command=command)
+                await self.activeDb.DeleteFromDb(command)
+                await ctx.send("Custom command successfully deleted.")
+            except NotFound:
+                await ctx.send("That command doesn't exist.")
+        # if user isnt an admin, only allows them to delete their own commands
+        else:
+            if self.activeDb.BelongsToUser(command, ctx.message.author.id) == False:
+                await ctx.send("Hey, that's not yours.")
+                return
+
+            try:
+                await self.commandobj.delete(ctx=ctx, command=command)
+                await self.activeDb.DeleteFromDb(command)
+                await ctx.send("Custom command successfully deleted.")
+            except NotFound:
+                await ctx.send("That command doesn't exist.")
+
+    @customcom.command(name="edit")
+    async def cc_edit(self, ctx, command: str.lower, *, text: str = None):
+        """Edit a custom command.
+
+        Example:
+            - `[p]customcom edit yourcommand Text you want`
+
+        **Arguments:**
+
+        - `<command>` The custom command to edit.
+        - `<text>` The new text to return when executing the command.
+        """
+        if ctx.message.author.top_role.permissions.administrator:
+            await ctx.send("Admin Override.")
+
+            try:
+                await self.commandobj.edit(ctx=ctx, command=command, response=text)
+                await ctx.send("Custom command successfully edited.")
+            except NotFound:
+                await ctx.send(
+                    (
+                        "That command doesn't exist. Use"
+                        + ctx.clean_prefix
+                        + " to add it."
+                    )
+                )
+            except ArgParseError as e:
+                await ctx.send(e.args[0])
+            except CommandNotEdited:
+                pass
+        else:
+            # blocks a user from editing a command that isnt theirs
+            if self.activeDb.BelongsToUser(command, ctx.message.author.id) == False:
+                await ctx.send("Hey, that's not yours.")
+                return
+
+            try:
+                await self.commandobj.edit(ctx=ctx, command=command, response=text)
+                await ctx.send("Custom command successfully edited.")
+            except NotFound:
+                await ctx.send(
+                    (
+                        "That command doesn't exist. Use"
+                        + ctx.clean_prefix
+                        + " to add it."
+                    )
+                )
+            except ArgParseError as e:
+                await ctx.send(e.args[0])
+            except CommandNotEdited:
+                pass
+
+    def GetHighestUserCommAllowance(self, member):
```

Comment:
nit: Here and in other places where there are function definitions, don't mix lower_case_with_underscores with CapitalizedWords.